db/db-challenge3/db-001/db-001.sql

```diff
@@ -0,0 +1,79 @@
+-- ユーザー情報シートに記載されているデータを登録すること
+-- 空項目はNULLとする
+
+-- id=1 admin
+INSERT INTO `users`(
+  `screen_name`, `email`, `password`, `introduce`, `company_phone_number`, `is_deleted`, `created_at`, `updated_at`
+  ) VALUES (
```

Comment:
左側にかっこ揃えるのがベターかと（修正任意、以降全体的に同じ指摘）

```suggestion
) VALUES (
```